definitions/ramda_v0.21.x/test_ramda_v0.21.x.js

```diff
@@ -0,0 +1,225 @@
+/* @flow */
+/*eslint-disable no-undef, no-unused-vars, no-console*/
+'use strict'
+const _ = require('ramda')
+function describe(tag, fn) {
+  fn()
+}
+
+function it(tag, fn) {
+  fn()
+}
+
+describe('Functions', () => {
```

Comment:
whoa, this is a cool idea!

I wonder if we could generalize the notion of using a runtime test runner + types for a better way of ensuring that our libdefs match up with their corresponding libraries...

cc @marudor @splodingsocks @ryyppy